raidpackage.py

```diff
@@ -123,10 +128,9 @@ async def __cancel_order(self, msg, usr_id):
     def __gracefully_complete_order(self, order: Order):
         del self.__responding_players[order.id]
 
-    async def __process_user_quantity_input(self, client, usr, item, qtyMax, usr_id):
-        __error_messages = []
+    async def __process_user_quantity_input(self, client, usr, item, qtyMax, usr_id, order: Order):
```

Comment:
Since we are passing `Order` we no longer need `qtyMax` and `usr` and `usr_id`

Reply:
Fixed 34da2ec